### PR TITLE
Optflow new message support

### PR DESCRIPTION
**Pull request by mhkabir**

### CODE REVIEW

File: mavros_extras/src/plugins/px4flow.cpp [hunk 1]
```diff
@@ -48,14 +51,22 @@ class PX4FlowPlugin : public MavRosPlugin {
...
-		else
-			flow_sub = nh.subscribe("optical_flow", 10, &PX4FlowPlugin::send_flow_cb, this);
+		flow_nh = ros::NodeHandle(nh, "px4flow");
+		
+		flow_nh.param<std::string>("frame_id", frame_id, "px4flow");
+
+		//Default rangefinder is Maxbotix HRLV-EZ4
+		flow_nh.param("ranger_type", ranger_type, 0);	// See sensor_msgs/Range docs 
+		flow_nh.param("ranger_fov", ranger_fov, 0.0); 	// TODO
+		flow_nh.param("ranger_min_range", ranger_min_range, 0.3);
```
Comment (mhkabir) [inline mavros_extras/src/plugins/px4flow.cpp]: @vooon Small TODO here, can you please check specifications and let me know what to fill in. I cannot find it in datasheet of SONAR.

Comment (vooon) [inline mavros_extras/src/plugins/px4flow.cpp]: In schematic says that it is MB1040: http://www.maxbotix.com/Ultrasonic_Sensors/MB1040.htm

File: mavros_extras/src/plugins/px4flow.cpp [hunk 2]
```diff
@@ -64,68 +75,82 @@ class PX4FlowPlugin : public MavRosPlugin {
...
+		
+		temp_pub.publish(temp_msg);
+
+		// Rangefinder
+		sensor_msgs::RangePtr range_msg = 
+			boost::make_shared<sensor_msgs::Range>();
+
+		range_msg->header = header;
+		
+		range_msg->radiation_type = ranger_type;
```
Comment (vooon) [inline mavros_extras/src/plugins/px4flow.cpp]: Radiation type should be ULTRASOUND, e.g `sensor_msgs::Range::ULTRASOUND`.

**Comment by vooon** [comment]
Looks good, but:
- fix header inclusion (travis)
- radiation type: Ultrasound

**Comment by mhkabir** [comment]
@vooon This is looking fine. Merging.

**Pull request merged**